tests/Fixtures/inc/Addon/GoogleTracking/GoogleTagManager/replaceUrl.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+return [
+	'vfs_dir' => 'wp-content/cache/',
+	'structure' => [
+		'wp-content' => [
+			'cache' => [
+				'busting' => [
+					'1' => []
+				],
+				'google-tracking' => []
+			]
+		]
+	],
+	'test_data' => [
+		'shouldBailOutWhenNoValidScript' => [
+			'config' => [
+				'html' => '<html><head><script async src="https://www.test.com/rocket.js"></script></head></html>',
+			],
+			'expected' =>
+				[
+					'unit' => '<html><head><script async src="https://www.test.com/rocket.js"></script></head></html>',
+					'integration' => '<html><head><script async src="https://www.test.com/rocket.js"></script></head></html>',
+				]
+		],
+		'shouldReplaceUrlWithRelativeProtocol' => [
+			'config' => [
+				'url' => 'https://www.googletagmanager.com/gtag/js?id=UA-135617916-1',
+				'html' => '<html><head><script async src="//www.googletagmanager.com/gtag/js?id=UA-135617916-1"></script></head></html>',
+			],
+			'expected' => [
+				'unit' => '<html><head><script data-no-minify="1" async src="vfs://public/wp-content/cache/busting/1/gtm-88c587e9d2fdeb7ac5d4cdd9bd8d4af5.js"></script></head></html>',
```

Comment:
Why do we need in expected both unit & integration? 
We can use a placeholder for the domain and / or wp-content  folderand replace it in test with the proper domain based on each test.
What do you think? Will this work?

Reply:
I thought that this will be easier for non developers to add fixture with full values, but your idea is totally right.

Reply:
@engahmeds3ed If there are no differences, keep it simple and use the same expected string for both test suites.

Reply:
Ok

Reply:
@crystinutzaa now it's resolved, plz review and approve changes.